code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -773,6 +746,113 @@
 	M.adjustOxyLoss(10 * removed * scale)
 	M.Weaken(10 * removed * scale)
 
+/datum/reagent/mental/tramadol
+	name = "Tramadol"
+	id = "tramadol"
+	description = "Tramadol is a very effective and addictive painkiller designed for victims of heavy physical trauma. Does not work when inhaled."
+	reagent_state = LIQUID
+	color = "#CB68FC"
+	overdose = 30
+	scannable = 1
+	metabolism = 0.02
+	goodmessage = list("You feel bulletproof.","You feel strong!.","You feel alert and focused.")
```

Comment:
As below, if we're going to roughly copypaste things, might as well use the things suggested below to enhance immersion.